### ISSUE TITLE
only worked on BLE4.0

### ISSUE THREAD

**Issue by 18811708533** [DigitalSecurity/btlejuice#39]

**Comment by 18811708533** [comment]
I install the project,and Use huawei band A1 、A3(BLE 4.2)  to do the experiment，but fail.I guess it's because huawei band A1 、A3 use  BLE 4.2 not BLE 4.0 ?  Am I correct?
![2018-07-13 11-18-25](https://user-images.githubusercontent.com/27751202/42670635-9fb1c828-868e-11e8-98d9-4c5a4ae36873.png)

**Comment by virtualabs** [comment]
Not necessarily, I managed to get it working with a stock BLE 4.2 adapter and a BLE 4.2 compatible device. Maybe you're experiencing some kind of application timeout ? Are you using the expected application to connect to the device ?
